nixos/tests/networking/networkd-and-scripted.nix

```diff
@@ -300,26 +184,23 @@ let
         '';
     };
     bond = let
-      node = address: { pkgs, ... }: with pkgs.lib; {
+      node = address: clientConfig {
         virtualisation.interfaces.enp1s0.vlan = 1;
         virtualisation.interfaces.enp2s0.vlan = 2;
         networking = {
-          useNetworkd = networkd;
-          useDHCP = false;
           bonds.bond0 = {
             interfaces = [ "enp1s0" "enp2s0" ];
             driverOptions.mode = "802.3ad";
           };
-          interfaces.bond0.ipv4.addresses = mkOverride 0
+          interfaces.bond0.ipv4.addresses = lib.mkOverride 0
```

Comment:
would it be possible to move these `with lib` changes into a separate pr that can be insta merged? or at least a separate commit? (can use git-revise, jujutsu, or similar to do that semi easily)

Reply:
see #296500